app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.jsx

```diff
@@ -29,7 +29,7 @@ function DocumentCaptureTroubleshootingOptions() {
             url: getHelpCenterURL({
               category: 'verify-your-identity',
               article: 'accepted-state-issued-identification',
-              location: 'troubleshooting_options',
+              location: 'document_capture_troubleshooting_options',
```

Comment:
See below, which this should have matched:

https://github.com/18F/identity-idp/blob/39730b2c5133204bae69a897ef6bcf6767f21aa9/app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.jsx#L38